.asf.yaml

```diff
@@ -52,3 +52,24 @@ github:
     squash: true
     merge: false
     rebase: false
+
+  protected_branches:
+    master:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: false
+        # contexts are the names of checks that must pass
+        contexts:
+          - Docker/build
+          - Frontend/build
+          - PR Lint/check
+          - E2E/Cypress
+          - lint
+          - test-mysql
+          - test-postgres
+          - test-sqlite
+
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
+        require_code_owner_reviews: true
+        required_approving_review_count: 1
```

Comment:
definitely would like a second pair of eyes on all this.

Reply:
The only risk I see is the contexts are not correct. Let's just be ready to fix/revert if needed

Reply:
Thanks, yeah, I set up: https://github.com/apache/superset/pull/12698 and will rebase when this lands. We might as well wait until next week, too, to land this.